test.js

```diff
@@ -137,7 +165,7 @@ test('support `Object.create(null)` objects', t => {
 test('support `Object.create(null)` targets', t => {
 	var obj = Object.create(null);
 	obj.foo = true;
-	t.same(fn(obj, {bar: false}), {foo: true, bar: false});
+	t.same(fn({}, obj, {bar: false}), {foo: true, bar: false});
```

Comment:
Nothing is wrong with this test. The first deep-assign argument is the target and this test asserts, that the following does not throw an Error:
```js
deep-assign(Object.create(null), …)
```